src/js_parser.zig

```diff
@@ -8286,11 +8650,7 @@ fn NewParser_(
                             else => {
                                 if (comptime get_metadata) {
                                     const find_result = p.findSymbol(logger.Loc.Empty, p.lexer.identifier) catch unreachable;
-                                    if (p.known_enum_values.contains(find_result.ref)) {
```

Comment:
What was this change?

Reply:
known_enum_values was removed in favor of real enum inlining.